proxy/src/redis/notifications.rs

```diff
@@ -74,7 +74,11 @@ pub(crate) enum Notification {
     #[serde(rename = "/cancel_session")]
     Cancel(CancelSession),
 
-    #[serde(other, skip_serializing)]
+    #[serde(
+        other,
+        deserialize_with = "deserialize_unknown_topic",
+        skip_serializing
+    )]
     UnknownTopic,
```

Comment:
Maybe `UnknownTopic(IgnoredAny)` works as well? Without any extra deser functions

Reply:
serde(other) wants a unit variant. Otherwise I would have put serde_json::(Raw))Value in there.

Reply:
Maybe dtolnay did this deliberately to keep the option open for a generic payload in the future.